crates/relayer-amplifier-api-integration/src/lib.rs

```diff
@@ -1,6 +1,9 @@
 //! The component that is responsible for communicating with the Axelar Amplifier API
 
-mod component;
+//! This crate provides integration with the Axelar Amplifier API, which is responsible for
+//! coordinating cross-chain message delivery and validation.
+/// The core Amplifier API abstraction.
+pub mod component;
```

Comment:
Aren't the types exported below enough? It seems we're exporting (almost?) everything there :thinking: